snippets/rust.snippets

```diff
@@ -93,70 +108,72 @@ snippet loop "loop {}" b
 	loop {
 		${0}
 	}
-snippet while
+snippet while "while loop"
 	while ${1:condition} {
 		${0}
 	}
-snippet for
+snippet for "for ... in ... loop"
 	for ${1:i} in ${2:0u..10} {
 		${0}
 	}
-snippet spawn
-	spawn(proc() {
-		${0}
-	});
-snippet chan
-	let (${1:tx}, ${2:rx}): (Sender<${3:int}>, Receiver<${4:int}>) = channel();
-snippet duplex
-	let (${1:from_child}, ${2:to_child}) = sync::duplex();
```

Comment:
The `duplex` function moved from `sync` and [is now deprecated][1] anyway, so I dropped this snippet entirely.

[1]: http://doc.rust-lang.org/0.12.0/std/comm/fn.duplex.html